### PR TITLE
Shayan/Fixing Form Dropdown and ColorPicker

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

File: sass/components/form.scss [hunk 1]
```diff
@@ -279,27 +285,27 @@
...
-        width: 100%;
+        z-index: 2;
         max-height: 200px;
         overflow: scroll;
         box-shadow: 0 2px 8px 0 rgba(42, 48, 82, 0.14);
         box-sizing: border-box;
         border-radius: 4px;
         font-size: 14px;
+        width: 272px;
+        transition: opacity 0.2s, top 0.2s;
```
Comment (zaki-binary) [inline sass/components/form.scss:300]: Same here, can we use `transform translate` instead?

File: sass/components/form.scss [hunk 2]
```diff
@@ -439,24 +457,24 @@
...
         right: -5px;
         opacity: 0;
         pointer-events: none;
-        z-index: +2;
+        z-index: 2;
         width: 266px;
         padding: 8px;
-        transition: opacity 0.1s;
         border-radius: 4px;
+        transition: opacity 0.2s, top 0.2s;
```
[the remark this answers is not in the document]
Comment (shayan-binary-2) [reply sass/components/form.scss]: replaced

File: sass/components/form.scss (hunk 1 above)
Comment (shayan-binary-2) [reply sass/components/form.scss]: replaced

File: src/store/SettingsDialogStore.js [hunk 3]
```diff
@@ -21,13 +22,39 @@ export default class SettingsDialogStore {
...
+            this.setFreezeScroll(false);
+        });
     }
 
     get context() { return this.mainStore.chart.context; }
     get stx() { return this.context.stx; }
     get theme() { return this.mainStore.chartSetting.theme; }
 
     @computed get open() { return this.menu.open; }
+    @action.bound checkDropdownOpen() {
```
Comment (balakrishna-binary) [inline src/store/SettingsDialogStore.js:45]: I don't think we need `@action.bound` decorator here as we don't modify any observables

Comment (shayan-binary-2) [reply src/store/SettingsDialogStore.js]: @balakrishna-binary yes you are right. just modified and updated the test link

**Pull request merged**